templates/federated_reporting/import.sh

```diff
@@ -100,6 +102,22 @@ else
   done
 fi
 
+if [ "$CFE_FR_HANDLE_DUPLICATES" = "yes" ]; then
```

Comment:
This new config variable should be added near the top of the script where it checks if all such variables are defined.